itests/org.openhab.core.config.discovery.tests/src/main/java/org/openhab/core/config/discovery/inbox/DynamicThingUpdateOSGiTest.java

```diff
@@ -121,7 +121,6 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
                 return thingHandler;
             }
         };
-        return thingHandlerFactory;
```

Comment:
I would prefer to keep the old version here, it look cleaner.

Reply:
ok, rolled back